src/controllers/user.controllers.ts

```diff
@@ -1,18 +1,24 @@
-import { Request, Response } from "express";
+import { NextFunction, Request, Response } from "express";
 import bcrypt from "bcrypt";
-import * as jwt from "jsonwebtoken";
+import jwt, { Jwt, JwtPayload } from "jsonwebtoken";
```

Comment:
Why we import jwt 2 times and also Jwtpayload ??

Reply:
fixed it